samples/shields/x_nucleo_iks01a2/sample.yaml

```diff
@@ -2,8 +2,7 @@ sample:
   name: X-NUCLEO-IKS01A2 sensor shield
 tests:
   sample.shields.x_nucleo_iks01a2:
-    platform_exclude: disco_l475_iot1
     harness: shield
     tags: shield
-    depends_on: arduino_i2c
+    depends_on: arduino_i2c arduino_header
```

Comment:
Is not one of them redundant? A board can not have arduino_i2c without arduino_header.

Reply:
I think arduino_header might be poorly named.  This might be better as arduino_gpio.

Reply:
Add a commit to rename the depend on to `arduino_gpio`